activerecord/lib/active_record/attribute_methods/time_zone_conversion.rb

```diff
@@ -98,11 +98,14 @@ def create_time_zone_conversion_attribute?(name, cast_type)
               still causes `String`s to be parsed as if they were in `Time.zone`,
               and `Time`s to be converted to `Time.zone`.
 
+              To silence this deprecation warning, add one of the following to your
+              initializer:
+
               To keep the old behavior, you must add the following to your initializer:
```

Comment:
Thanks for the PR! These two lines are repetitive. I think the new line should be removed and that the next change you made should read "To use the new behavior, add the following:"

Reply:
wow, thanks for responding so quickly! The changes have been made :)